src/pages/index.js

```diff
@@ -32,6 +32,43 @@ const formatString = (str) => {
     return str.toLowerCase().split(" ").join('-');
 }
 
+function hash(s){
+  return s.split("").reduce(function(a,b){a=((a<<5)-a)+b.charCodeAt(0);return a&a},0);              
+}
+
+function hideOverlay() {
+  let overlay = document.querySelector(".lockout-overlay");
+  overlay.classList.add("hidden");
+  let layout = document.querySelector(".layout");
+  layout.classList.remove("blurred");
+  window.scrollTo(0,0);
+}
+
+function showOverlay() {
+  let overlay = document.querySelector(".lockout-overlay");
+  overlay.classList.remove("hidden");
+  let layout = document.querySelector(".layout");
+  layout.classList.add("blurred");
+  window.scrollTo(0,0);
+}
+
+function unlock(e) {
+    e.preventDefault();
+    var password = document.getElementById("password").value;
+
+    if(hash(password) === 1032675042) {
```

Comment:
not super secure, if someone wants to log in and look at the javascript... but at least it's hashed..

just trying to keep out the riff-raff